scipy/optimize/tests/test_linprog.py

```diff
@@ -338,11 +338,11 @@ def f(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None, bounds=None):
             linprog(c, A_ub, b_ub, A_eq, b_eq, bounds,
                     method=self.method, options=self.options)
 
-        for bad_bound in [[(5, 0), (1, 2), (3, 4)],
-                          [(1, 2), (3, 4)],
+        # Removed [(5, 0), (1, 2), (3, 4)]: these are invalid bounds but should be subject to a check in _presolve, not in _clean_inputs.
+        # The optimization should exit with an 'infeasible problem' error, not with a ValueError
```

Comment:
I think they should be added back as tests, then, except that this time the function should return an `OptimizeResult` object with infeasible status.

Reply:
I'll note this for the update of the `_presolve` routine.

Reply:
OK.